src/types/inp.ts

```diff
@@ -116,6 +150,26 @@ export interface INPAttribution {
    * (e.g. usually in the `dom-interactive` phase) it can result in long delays.
    */
   loadState: LoadState;
+  /**
+   * The total duration of Long Animation Frame scripts that intersect the INP
+   * duration. Note, this is limited to scripts > 5 milliseconds.
+   */
+  scriptsDuration?: number;
+  /**
+   * The total forced style and layout duration from any
+   * Long Animation Frames intersecting INP interaction.
+   */
+  scriptsForcedStyleAndLayoutDuration?: number;
```

Comment:
Similar to the above, I wonder if this should be called `totalForcedStyleAndLayoutDuration`.

Reply:
Done.